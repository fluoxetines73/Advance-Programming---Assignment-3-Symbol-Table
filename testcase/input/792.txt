BEGIN
END
RPRINT
PRINT
BEGIN
RPRINT
INSERT x string
LOOKUP x
BEGIN
INSERT d string
LOOKUP x
END
END