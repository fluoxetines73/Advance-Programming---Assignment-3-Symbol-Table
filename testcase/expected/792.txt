


success
1
success
1